pkg/tso/keyspace_group_manager.go

```diff
@@ -72,14 +72,14 @@ type state struct {
 	// keyspaceLookupTable is a map from keyspace to the keyspace group to which it belongs.
 	keyspaceLookupTable map[uint32]uint32
 	// splittingGroups is the cache of splitting keyspace group related information.
-	splittingGroups map[uint32]struct{}
+	splittingGroups map[uint32]time.Time
```

Comment:
How about adding a comment to indicate what it stores?

Reply:
Addede.